docs/install_csi_driver.md

```diff
@@ -0,0 +1,174 @@
+# Nutanix CSI Driver installation with CAPX
+
+The Nutanix CSI driver is fully supported on CAPI/CAPX deployed clusters where nodes follow Nutanix prerequisites.
+
+You can install it yourself in a workload cluster by following the official documentation (Helm based) or let the management cluster install it for you using `ClusterResourceSet`.
+
+
+
+## CAPI Workload cluster prerequisites for Nutanix CSI Driver
+
+Kubernetes workers need the following prerequisite to use the Nutanix CSI Drivers:
+
+- iSCSI initiator package (for Volumes based block storage)
+- NFS client package ( for Files based storage)
+
+These packages can already be present in the image you use with your Infrastructure provider or you can also rely on your bootstrap provider to install them.
```

Comment:
can we rephrase it without assumptions of package being present or user action is needed? how does one verify it?